modules/ROOT/pages/platform/metrics-integration.adoc

```diff
@@ -0,0 +1,430 @@
+
+# example output
+{
+  endpoint: "https://customer-metrics-api.neo4j.io/api/<YOUR_TENANT_ID>/metrics"
+}
+
+# extract endpoint
+aura tenants get-metrics-integration --tenant-id <YOUR_TENANT_ID> | jq '.endpoint'
+----
+
+* For instance metrics endpoint, Aura CLI `instances get` command JSON output includes a new property `metrics_integration_url`:
++
+----
+aura instances get --instance-id <YOUR_INSTANCE_ID>
+
+# example output
+{
+    "id": "id",
+    "name": "Production",
+    "status": "running",
+    "tenant_id": "YOUR_TENANT_ID",
+    "cloud_provider": "gcp",
+    "connection_url": "YOUR_CONNECTION_URL",
+    "metrics_integration_url": "https://customer-metrics-api.neo4j.io/api/<YOUR_TENANT_ID>/<YOUR_INSTANCE_ID>/metrics",
+    "region": "europe-west1",
+    "type": "enterprise-db",
+    "memory": "8GB",
+    "storage": "16GB"
+  }
+
+# extract endpoint
+aura instances get --instance-id <YOUR_INSTANCE_ID> | jq '.metrics_integration_url'
+----
+
+* Reference: https://neo4j.com/labs/aura-cli/1.0/cheatsheet/[Aura CLI cheetsheet^]
+
+[#metric_definitions]
+== Metric Definitions
+
+|===
+| Title | Description | MetricName | MetricType | DefaultAggregation
+
+| Out of Memory Errors
+| The total number of Out of Memory Errors for the instance. Consider increasing the size of the instance if any OOM errors.
+| `neo4j_aura_out_of_memory_errors_total`
+| _Counter_
+| SUM
+
+| CPU Available
+| The total CPU cores assigned to the instance nodes.
+| `neo4j_aura_cpu_limit`
+| _Gauge_
+| MAX
+
+| CPU Usage
+| CPU usage (cores). CPU is used for planning and serving queries. If this metric is constantly spiking or at its limits, consider increasing the size of your instance.
+| `neo4j_aura_cpu_usage`
+| _Gauge_
+| MAX
+
+| Memory Used
+| Memory in use by Neo4j.
+| `neo4j_aura_memory_usage`
+| _Gauge_
+| MAX
+
+| Memory Total
+| The total memory assigned to the instance.
+| `neo4j_aura_memory_limit`
+| _Gauge_
+| MAX
+
+| Storage Total
+| The total disk storage assigned to the instance.
+| `neo4j_aura_storage_limit`
+| _Gauge_
+| MAX
+
+| Heap Used
+| The percentage of configured heap memory in use. The heap space is used for query execution, transaction state, management of the graph etc. +
+The size needed for the heap is very dependent on the nature of the usage of Neo4j. For example, long-running queries, or very complicated queries, are likely to require a larger heap than simpler queries. Generally speaking, in order to aid performance, the heap should be large enough to sustain concurrent operations. +
+This value should not exceed 80% for long periods of time, short spikes can be normal. In case of performance issues you may have to tune your queries, and monitor their memory usage, in order to determine whether the heap needs to be increased. If the workload of Neo4j and performance of queries indicates that more heap space is required, consider increasing the size of your instance. This will help avoid unwanted pauses for garbage collection.
```

Comment:
```suggestion
This value should not exceed 80% for long periods, short spikes can be normal. 
In case of performance issues, you may have to tune your queries and monitor their memory usage, to determine whether the heap needs to be increased. 
If the workload of Neo4j and performance of queries indicates that more heap space is required, consider increasing the size of your instance. 
This helps avoid unwanted pauses for garbage collection.
```